docs/parameter-types.md

```diff
@@ -0,0 +1,36 @@
+# Parameter types
+
+Command body structures for [SetParameters](./commands/SetParameters.md).
+Attention! Parameter types is not fully documented!
+
+
+## Reporting data type
+
+Parameter of that type used to setup type of values received from device.
+
+### Format
+
+> Size | Type | Field
+> -----|------|-------
+> 1    | byte | type = `5`
+> 1    | byte | [type](#data-type)
+
+#### **data type**
```

Comment:
either "type" here or "data type" in format

Reply:
`"data type" in format` - done